README.md

```diff
@@ -247,6 +248,49 @@ User.query("ruby")
 For more information on index types and their options, see the [CipherStash
 docs](https://docs.cipherstash.com/reference/index-types/index.html).
 
+## Unique indexes
+
+ActiveStash supports adding server side unique constraints on `:range` and `:exact` indexes.
+
+Unique constraints are not supported on `:match` indexes.
+
+To specify a unique constraint on a field add `unique: true`.
+
+In the below example a unique constraint is added to the email field.
+
+This will result in a `:match` index and a unique constraint on the `:range` and `:exact` indexes.
+
+```ruby
+class User < ActiveRecord::Base
+  include ActiveStash::Search
+
+  stash_index :name, :dob
+  stash_index :email, unique: true
+
+  encrypts :name
+  encrypts :email
+  encrypts :dob
+end
+```
+
+Used in combination with `except` and `only` you can specify a unique constraint on a
```

Comment:
My thinking was to show the syntax for adding a unique constraint when using except and only. So if they didn't want the other indexes, like range for ordering or match for free text search this is what it would look like.

Do you think this is unnecessary info?